.gitignore

```diff
@@ -4,3 +4,9 @@
 build/
 dist/
 *.egg-info/
+/.coverage
+/coverage.xml
+
+# VS Code
```

Comment:
sorry, but no coder specific stuff here.

I case we add we have to support all editors/IDE stuff.

Try to add your files to be ignored to your ~/.gitconfig file like:

```
[core]                                                                                              
    excludesfile = /home/hthiery/.gitignore_global
```
and edit your personal ignore list in .gitignore_global
```
 ctags.txt
 *.bak
 *.swp
 *.prj
```